.travis.yml

```diff
@@ -4,7 +4,7 @@ os:
 language: go
 
 go:
-  - 1.11.x
+  - 1.14.x
```

Comment:
@hannahhoward it seems like we need to be rebased on master since we're not using travis anymore.